includes/wf_crm_webform_postprocess.inc

```diff
@@ -1779,6 +1779,13 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
       // DRU-2862396 - we must ensure to only present 2 decimals to CiviCRM:
       $contributionRecurAmount = floor(($contributionParams['total_amount'] / $installments) * 100) / 100;
       $contributionFirstAmount = $contributionParams['total_amount'] - $contributionRecurAmount * ($installments - 1);
+      $salesTaxFirstAmount = $contributionParams['tax_amount'] - floor(($contributionParams['tax_amount'] / $installments) * 100) / 100 * ($installments - 1);
+    }
+    // Calculate the line_items for the contributionFirst:
+    foreach ($this->line_items as $key => $k) {
+      $this->line_items[$key]['unit_price'] = $k['unit_price'] / $installments;
+      $this->line_items[$key]['line_total'] = $k['line_total'] / $installments;
+      $this->line_items[$key]['tax_amount'] = $k['tax_amount'] / $installments;
```

Comment:
Hmm, reading the code it looks like on line 1778 `$installments` might be zero. But this foreach loop runs unconditionally, which could give us a divide by zero error?

Reply:
Yes that already happens before -> line 1754:
if ($numInstallments != 1 && !empty($frequencyInterval)) {

Reply:
0 is a special case that means don't do installments - but start a recurring series - open ended commitment;